lib/provider/vpc_data_types.go

```diff
@@ -26,13 +26,15 @@ type VPCVolume struct {
 	VolumeEncryptionKey *VolumeEncryptionKey `json:"encryption_key,omitempty"`
 	Profile             *Profile             `json:"profile,omitempty"`
 	CRN                 string               `json:"crn,omitempty"`
+	ETag                string               `json:"etag,omitempty"`
```

Comment:
where you are using ETag, I don't think so we need ETag. Can you check

Reply:
@arahamad Check the code in library , we need to store it in models.volume which gets converted to provider.volume in GETVolume call 

https://github.com/IBM/ibmcloud-volume-vpc/pull/78/files#diff-4f2cc52b96957a217c6cb44cbf79ef9ca14939e6a8019bba1c02ad2ce87dc115R63

https://github.com/IBM/ibmcloud-volume-vpc/pull/78/files#diff-745dcaa283266cb8ef2e17fc7d58afe3dc142129bdb0d022a640eb397ba7da6bR50